app/src/features/projects/view/components/LocationBoundary.tsx

```diff
@@ -125,12 +125,11 @@ const LocationBoundary: React.FC<ILocationBoundaryProps> = (props) => {
   };
 
   useEffect(() => {
-    const geometryCollectionResult = generateValidGeometryCollection(location.geometry);
-    const nonEditableGeometriesResult = geometryCollectionResult.geometryCollection.map((geom: Feature) => {
+    const nonEditableGeometriesResult = location.geometry.map((geom: Feature) => {
       return { feature: geom };
     });
 
-    setBounds(geometryCollectionResult.bounds);
+    updateMapBounds(location.geometry, setBounds);
```

Comment:
Wondering about this function, any reason for passing in the setBounds function, and not just returning the bounds from this function, and then calling setBounds here? In this case, the function would become something like `getMapBounds(geometry: Feature[]): number[][]` (not sure on the exact types)

Reply:
ok cool